File: libs/esl/perl/single_command.pl
#!/usr/bin/perl
require ESL;

my $command = shift;
my $args = join(" ", @ARGV);

my $con = new ESL::ESLconnection("127.0.0.1", "8021", "ClueCon");
my $e = $con->api($command, $args);
print $e->getBody();

File: libs/esl/ruby/single_command.rb
#! /usr/bin/ruby
require 'ESL' 

HOST     = '127.0.0.1'.to_s 
PORT     = '8021'.to_s 
PASSWORD = 'ClueCon'.to_s 

command = ARGV.join(" ")

con = ESL::ESLconnection.new(HOST,PORT,PASSWORD) 
e = con.sendRecv('api ' + command)

puts e.getBody

